test/route-install.js

```diff
@@ -46,15 +67,20 @@ describe('install route', () => {
       ];
 
       this.pkgData = {
-        author: {
-          name: 'Simon'
+        'dist-tags': {
```

Comment:
The latest `package-json` expects these properties: https://github.com/sindresorhus/package-json/blob/2312d0437c16d7117d354ae76f640c5ac25fe0e0/index.js#L39-L56